src/battle_util.c

```diff
@@ -8862,6 +8862,9 @@ static inline u32 CalcMoveBasePower(u32 move, u32 battlerAtk, u32 battlerDef, u3
     case EFFECT_MAX_MOVE:
         basePower = GetMaxMovePower(gBattleMons[battlerAtk].moves[gBattleStruct->chosenMovePositions[battlerAtk]]);
         break;
+    case EFFECT_RAGE_FIST:
+        basePower += 50 * gBattleStruct->timesGotHit[gBattlerPartyIndexes[battlerAtk]][GetBattlerSide(battlerAtk)];
```

Comment:
The damage should be capped at 350

Reply:
The counter is limited at 6, so it is accounted for

Reply:
The damage and not the counter should be limited. If the field ever needs to be reused in the future and whatever is using it has a different cap, the field value can always be relied on.

Reply:
Yes, this makes sense, will adjust.